tsl/test/expected/dist_triggers.out

```diff
@@ -0,0 +1,511 @@
+CREATE TRIGGER z_test_trigger_all_after
+    AFTER INSERT OR UPDATE OR DELETE ON hyper
+    FOR EACH ROW EXECUTE FUNCTION test_trigger();
+-- statement triggers: BEFORE
+CREATE TRIGGER _0_test_trigger_insert_s_before
+    BEFORE INSERT ON hyper
+    FOR EACH STATEMENT EXECUTE FUNCTION test_trigger();
+CREATE TRIGGER _0_test_trigger_update_s_before
+    BEFORE UPDATE ON hyper
+    FOR EACH STATEMENT EXECUTE FUNCTION test_trigger();
+CREATE TRIGGER _0_test_trigger_delete_s_before
+    BEFORE DELETE ON hyper
+    FOR EACH STATEMENT EXECUTE FUNCTION test_trigger();
+-- statement triggers: AFTER
+CREATE TRIGGER _0_test_trigger_insert_s_after
+    AFTER INSERT ON hyper
+    FOR EACH STATEMENT EXECUTE FUNCTION test_trigger();
+CREATE TRIGGER _0_test_trigger_update_s_after
+    AFTER UPDATE ON hyper
+    FOR EACH STATEMENT EXECUTE FUNCTION test_trigger();
+CREATE TRIGGER _0_test_trigger_delete_s_after
+    AFTER DELETE ON hyper
+    FOR EACH STATEMENT EXECUTE FUNCTION test_trigger();
+SELECT * FROM create_distributed_hypertable('hyper', 'time', chunk_time_interval => 10);
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             1 | public      | hyper      | t
+(1 row)
+
+--test triggers before create_distributed_hypertable
+INSERT INTO hyper(time, device_id,sensor_1) VALUES
+(1257987600000000000, 'dev1', 1);
+-- Show trigger count on access node. Only statement-level triggers
+-- fire on the access node.
+SELECT tg_when, tg_level, tg_op, tg_name, count(*)
+FROM trigger_events
+GROUP BY 1,2,3,4
+ORDER BY 1,2,3,4;
+ tg_when | tg_level  | tg_op  | tg_name | count 
+---------+-----------+--------+---------+-------
+ AFTER   | STATEMENT | INSERT |         |     1
+ BEFORE  | STATEMENT | INSERT |         |     1
+(2 rows)
+
+-- Show trigger counts on data nodes. Both statement-level and
+-- row-level triggers fire on the data nodes.
+SELECT * FROM test.remote_exec(ARRAY[:'DATA_NODE_1', :'DATA_NODE_2', :'DATA_NODE_3'], $$
+SELECT tg_when, tg_level, tg_op, tg_name, count(*)
+FROM trigger_events
+GROUP BY 1,2,3,4
+ORDER BY 1,2,3,4;
+$$);
+NOTICE:  [db_dist_triggers_1]: 
+SELECT tg_when, tg_level, tg_op, tg_name, count(*)
+FROM trigger_events
+GROUP BY 1,2,3,4
+ORDER BY 1,2,3,4
+NOTICE:  [db_dist_triggers_1]:
+tg_when|tg_level |tg_op |tg_name                               |count
+-------+---------+------+--------------------------------------+-----
+AFTER  |ROW      |INSERT|_0_test_trigger_insert_after          |    1
+AFTER  |ROW      |INSERT|_0_test_trigger_insert_after_when_dev1|    1
+AFTER  |ROW      |INSERT|z_test_trigger_all_after              |    1
+AFTER  |STATEMENT|INSERT|_0_test_trigger_insert_s_after        |    1
+BEFORE |ROW      |INSERT|_0_test_trigger_insert                |    1
+BEFORE |ROW      |INSERT|z_test_trigger_all                    |    1
+BEFORE |STATEMENT|INSERT|_0_test_trigger_insert_s_before       |    1
+(7 rows)
+
+
+NOTICE:  [db_dist_triggers_2]: 
```

Comment:
Why don't we see the firing of the triggers on DN2 and DN3? Because rows didn't get inserted in them? Might be a good idea to add inserts which insert into these datanodes as well.